src/seedu/addressbook/tagging/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.tagging;
+
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.person.Name;
+
+/**
+ * Association class for classes Tag and Person.
+ * Each Tagging object will represent an adding or deleting of a tag for a specific person. 
+ */
+public class Tagging {
+	
+	private final static char ADD_PREFIX = '+';
+	private final static char DELETE_PREFIX = '-';
+	
+	private boolean addNotDelete;
+	private Name name;
+	private Tag tag;
+	
+	public Tagging(boolean addNotDelete, Name name, Tag tag) {
```

Comment:
Tagging is describing the association between Person and Tag. Use a Person object instead of Name